.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,40 @@
+name: deploy
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    environment: production
+    steps:
+    - name: Azure Service Principal CLI login
+      uses: azure/login@v2
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+    - name: Configure Docker
+      uses: docker/setup-buildx-action@v3
+
+    - name: Azure ACR login
+      uses: azure/docker-login@v2
+      with:
+        login-server: cfaprdconfigvalidation.azurecr.io
```

Comment:
what is a `login-server`?

Reply:
This is just the URL for the container registry where we are pushing images. You can see in the Azure portal under "Container Registries"